client/js/views/GameView.vue

```diff
@@ -294,39 +294,6 @@
       >
         <h1>Opponent Playing from Deck</h1>
       </v-overlay>
-      <counter-dialog
-        v-model="showCounterDialog"
-        :one-off="game.oneOff"
-        :target="game.oneOffTarget"
-        :twos-in-hand="twosInHand"
-        :twos-played="twosPlayed"
-        @resolve="resolve"
-        @counter="counter($event)"
-      />
-      <cannot-counter-dialog
-        v-model="showCannotCounterDialog"
-        :one-off="game.oneOff"
-        :opponent-queen-count="opponentQueenCount"
-        :player-two-count="playerTwoCount"
-        :twos-played="twosPlayed"
-        :target="game.oneOffTarget"
-        @resolve="resolve"
-      />
-      <four-dialog v-model="discarding" @discard="discard" />
-      <three-dialog
-        v-model="pickingFromScrap"
-        :one-off="game.oneOff"
-        :scrap="scrap"
-        @resolveThree="resolveThree($event)"
-      />
-      <seven-double-jacks-dialog
-        v-model="showSevenDoubleJacksDialog"
-        :top-card="topCard"
-        :second-card="secondCard"
-        @resolveSevenDoubleJacks="resolveSevenDoubleJacks($event)"
-      />
-      <game-over-dialog v-model="gameIsOver" :player-wins="playerWins" :stalemate="stalemate" />
-      <reauthenticate-dialog v-model="mustReauthenticate" />
```

Comment:
All of these now live in the GameDialogs component. The only exception is `reauthenticate-dialog` since it is also used at the top of GameView with in if/else. I wonder if it might be worth abstracting that later to an app-wide reauthentication.

Reply:
It very possibly would, though app wide Reauthentication would have more conditional logic

Reply:
I'd like to see that reauthentication getter and setter moved to a dispatches action so it's easier to share between components at least.